WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationStepView.swift

```diff
@@ -17,17 +17,14 @@ class MigrationStepView: UIView {
     private let centerView: UIView
     private let actionsView: MigrationActionsView
 
-    private lazy var centerContentView: UIView = {
-        let view = UIView()
-        view.addSubview(centerView)
-        return view
-    }()
```

Comment:
This view was previously added just to have leading and trailing margins for the `centerView`. We could achieve the same without this. [See `mainStackView` implementation.](https://github.com/wordpress-mobile/WordPress-iOS/pull/19944/files#diff-0a1685b35847103b8ed57f8ae8ebeee09ec41539f7e55fbb37e876c7b0231d8cR24)